docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -205,6 +209,16 @@ flavors have faster CPUs and no overcommitment of CPU cores.
 
 **cPouta HPC flavor characteristics:**
 
+**hpc.5.\*:**
+
+-   No redundant power
+-      CPU:   AMD EPYC 7702 64-Core Processor,
+    ***hyper-threading***
+-   Network: Redundant 25 Gb/s
+-   Flavor disk: Stored in the central storage
+-   Single-node or disk failures may cause downtime, but instances
+    are recoverable.
+
```

Comment:
These are Raid1 backed. Shouldnt cause downtime in this case.

Also these are without hyperthreading.